pkg/wkspace/minimal.go

```diff
@@ -120,8 +120,17 @@ func getValues(name string) (map[string]interface{}, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	backup, err := templateVals(name, path)
```

Comment:
Shouldn't we bail from this if `err != nil`?

Reply:
we override the original values with inputs from terraform, later we have to revert this to the original one. We can do this when there are no errors. Generally, this is the legacy code I accidentally removed